tools/importer/src/main/java/com/vesoft/nebula/tools/Options.java

```diff
@@ -0,0 +1,298 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.tools;
+
+import com.google.common.collect.Lists;
+import com.google.common.net.HostAndPort;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.kohsuke.args4j.Option;
+
+
+public class Options {
+    @Option(name = "-a", aliases = "--address", usage = "thrift service addresses. ip:port",
+            required = true)
+    String addresses;
```

Comment:
We can set some default values for user to simplify usage in command line. for example:
- address=127.0.0.1:3699; 
- user=user; 
- password=password;